webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -187,8 +202,12 @@
       </div>
 
       <div class="col-sm-8 detail-actions right-pane" *ngIf="currentTab === 'contacts'">
-        <div class="actions dropup" *ngIf="showActionBar && selectedContactDoc && !loadingContent && snapshotData?.name === 'contacts.detail'">
-          <span>
+        <div
+          mmAuth [mmAuthAny]="['can_edit', 'can_view_call_action', 'can_view_message_action']"
```

Comment:
Same here.

Reply:
Phone calls and sms messages in mobile open the phone and sms apps in the phone. It's not modifying the `medic` db. I think these 2 aren't strictly depending on `can_edit`

Reply:
Ahh I see. It's hard to see what's going on in this file. Needs a serious refactor, probably around the same time as FABs.